tools/staramr/staramr_search.xml

```diff
@@ -286,6 +326,17 @@
                 <element name="pointfinder_16S-rc_gyrA-rc_beta-lactam.fsa" file="test1-hits/pointfinder_16S-rc_gyrA-rc_beta-lactam.fsa" />
             </output_collection>
         </test>
+        <test>
+            <param name="genomes" value="16S-rc_gyrA-rc_beta-lactam.fsa" />
+            <param name="use_pointfinder" value="custom" />
+            <param name="pointfinder_custom" value="escherichia_coli" />
```

Comment:
I think you should have `ftype` here for the file(s).

https://docs.galaxyproject.org/en/latest/dev/schema.html#testing